src/Commands/ProjectCreateCommand.php

```diff
@@ -355,6 +437,12 @@ function ($state) use ($site_name, $label, $team, $target, $siteDir, $region) {
             ->progressMessage('Make initial commit')
             ->addCode(
                 function ($state) use ($siteDir, $source) {
+                    if (file_exists("$siteDir/web/modules/custom/.gitkeep")) {
```

Comment:
We shouldn't force-add like this. The reason these directories are .gitignore'd is that there is a Composer project type "Drupal custom module" which places projects in web/modules/custom. Folks can change the .gitignore if they want to commit files here instead. Is there a reason you need this committed to the repo?

Reply:
I need the folders to exist to run phpcs:
```
./vendor/bin/phpcs --standard=Drupal --extensions=php,module,inc,install,test,profile,theme,css,info,txt,md --ignore=node_modules,bower_components,vendor ./web/modules/custom
```

With this, I'm just force-adding an empty file in order to have the custom folder, but every other file in that folder will still be ignored to follow the current pattern.

I could also make a PR to https://github.com/pantheon-systems/drupal-project/blob/default/.gitignore to exclude the .gitkeep file and avoid force adding like this.

Another option would be to change the phpcs command to run it only if the folder exists.

Please let me know what do you think it's the best option and I'll update the code accordingly.